net/https-dns-proxy/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=https_dns_proxy
+PKG_VERSION:=2016-05-08e
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
```

Comment:
`PKG_RELEASE` should better be something "version-sortable".  We increase it when changes are made to the packaging method and reset it to initial value when `PKG_VERSION` is bumped